app/addons/qwintry/addon.xml

```diff
@@ -40,7 +40,8 @@
 			Phone 858-633-6353
 			5. Use logistics.qwintry.com control panel to attach preship tracking
 			*Feel free to call us +1-844-QWINTRY or send your questions to logistics@qwintry.com and we will be happy to assist!*</item>
-    </language_variables>
+		<item lang="EN" id="qwintry_wrong_measurement">Incorrect weight measurement, set in your http://logistics.qwintry.com account weight measurement - 	lb</item>
```

Comment:
Incorrect weight measurement settings were detected in your Qwintry Logistics profile, please change this to pounds (LB) in your http://logistics.qwintry.com account!